src/applications/ezr/tests/e2e/helpers/household.js

```diff
@@ -40,3 +40,145 @@ export const advanceFromHouseholdToReview = () => {
 
   goToNextPage('review-and-submit');
 };
+
+// Household section helper functions for spouse information
+export const fillSpousePersonalInformation = (spouseData = {}) => {
+  const {
+    firstName = 'John',
+    lastName = 'Doe',
+    ssn = '123422222',
+    dateOfBirth = '1990-1-1',
+    dateOfMarriage = '2015-6-15',
+  } = spouseData;
+
+  // Fill spouse full name
+  cy.fillVaTextInput('root_spouseFullName_first', firstName);
+  cy.fillVaTextInput('root_spouseFullName_last', lastName);
+  cy.fillVaTextInput('root_spouseSocialSecurityNumber', ssn);
+
+  // Filling using fillDateWebComponentPattern causes Cypress to change
+  // pages/introduces 'Leave site?' alert, but only on local. Use the
+  // following code instead.
+
+  // Fill spouse date of birth.
+  cy.get('va-memorable-date[name="root_spouseDateOfBirth"]')
```

Comment:
This whole block can be replaced with:

`cy.fillDate('root_spouseDateOfBirth', dateOfBirth);`